Cargo.toml

```diff
@@ -37,6 +37,7 @@ hex = "0.3"
 md5 = "0.6"
 decimal = { version = "2.0.4", default_features = false, optional = true }
 base64 = "0.12.1"
+lazy_static = "1.4.0"
```

Comment:
Since this is only needed for the tests, we can put it under `dev-dependencies` rather than `dependencies` so that users of the library don't need to pull it in.

Reply:
makes sense. fixed